commit f5791d1d7ec5e51bb093200745a5fc91dc1d7c4d (refs/changes/00/4200/3)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-06-13T17:58:16+00:00 (1 year, 4 months ago)
    
    MANTA-3758 ops zone missing reverse lookup data in binder
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/server.js b/lib/server.js
index 09272ed..078b4d3 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -302,6 +302,7 @@ function resolve(options, query, cb) {
         case 'load_balancer':
         case 'moray_host':
         case 'redis_host':
+        case 'ops_host':
         case 'rr_host':
                 addr = record[record.type].address;
                 query.addAnswer(domain, new ARecord(addr), ttl);
diff --git a/lib/zk.js b/lib/zk.js
index b7b998f..655efb2 100644
--- a/lib/zk.js
+++ b/lib/zk.js
@@ -175,6 +175,7 @@ TreeNode.prototype.onDataChanged = function (zk, data, stat) {
         case 'load_balancer':
         case 'moray_host':
         case 'redis_host':
+        case 'ops_host':
         case 'rr_host':
                 var record = parsedData[parsedData.type];
                 if (typeof (record) !== 'object' || record === null)
